Ex5Pt2.sh

```diff
@@ -0,0 +1,12 @@
+#Creates a modified version of wages.csv to include only gender, yearsExperience, wage sorted by wage
```

Comment:
1.	I’m taking .1 off here but this applies to the assignment as a whole. The prompt asks for a single shell script that answers all 3 questions and you’ve provided a different script for each answer
a.	0.9
2.	Your code works and produces 2 out of 3 answers correctly. Your answer for highest earner should be male	5	11	39.80892
a.	0.9
3.	Good code but you present the difference between the average wage of workers with 12 or 16 years of schooling while the question is looking for the difference in the minimum
a.	0.9